src/WattleScript.Interpreter/Tree/Statement.cs

```diff
@@ -316,8 +316,8 @@ private static Statement DispatchForLoopStatement(ScriptLoadingContext lcontext)
 					return new ForLoopStatement(lcontext, name, forTkn, false);
 				default:
 				{
-					if (CheckRangeFor(lcontext))
-						return new ForRangeStatement(lcontext, name, forTkn, paren);
+					//if (CheckRangeFor(lcontext))
```

Comment:
If removing ForRangeStatement, that should be done in this PR as opposed to commenting out

Reply:
Removed the comment here, if it's ok with you I'll remove ForRangeStatement itself in the follow up PR that will improve iterating perf. I've marked ForRangeStatement obsolete for now and keeping it around until then as a reference for me

Reply:
That's ok for now